Libplanet/Store/IStateStore.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.Collections.Immutable;
+using System.Security.Cryptography;
+using Bencodex.Types;
+using Libplanet.Action;
+using Libplanet.Blocks;
+
+namespace Libplanet.Store
+{
+    public interface IStateStore
+    {
+        void SetStates(HashDigest<SHA256> blockHash, IImmutableDictionary<string, IValue> states);
+
+        IValue GetState(
+            string stateKey,
+            HashDigest<SHA256>? blockHash = null,
+            Guid? chainId = null);
+
+        /// <summary>
+        /// Check the states corresponded to the block derived from <paramref name="blockHash"/>.
+        /// </summary>
+        /// <param name="blockHash">The <see cref="Block{T}.Hash"/> of <see cref="Block{T}"/>.
+        /// </param>
+        /// <returns>Whether to exist the states corresponded to <paramref name="blockHash"/>.
+        /// </returns>
+        bool BlockStateExists(HashDigest<SHA256> blockHash);
```

Comment:
How about `ExistsBlockState`?

Reply:
It seems better, I will apply it. Thanks! 🙏

Reply:
Applied in 1cf3a85.